server/handlers.js

```diff
@@ -144,4 +155,212 @@ const userFetchHandler = async (req, res) => {
     })
 }
 
-module.exports = {signinHandler, signupHandler, postHandler, readHandler, userFetchHandler};
\ No newline at end of file
+const changeAvatarHandler = async (req, res) => {
+    const {imgURL, user_id} = req.body;
+    await client.connect();
+    const db = client.db('pang');
+    await db.collection('users').update({_id: user_id}, {$set : {avatar:imgURL}}, (err, result) => {
+        if (result) {
+            res.status(200).json({status: 200, data: result})
+        } else {
+            res.status(400).json({status: 400, message:'ERROR'})
+        }
+    })
+}
+
+const followHandler = async (req, res) => {
+    const _id = req.params._id;
+    const userId = req.params.userId;
+
+    await client.connect();
+    const db = client.db('pang');
+    await db.collection('users').update({_id: _id}, {$push : { followedBy : userId}}, (err, result) => {
+        if (result) {
+            db.collection('users').update({_id: userId}, {$push : { following : _id}}, (err, result) => {
+                if (result) {
+                    res.status(200).json({status:200, data: result})
+                } else {
+                    res.status(400).json({status: 400, message: 'User Not Found'})
+                }
+            })
+        } else {
+            res.status(400).json({status: 400, message: 'User Not Found'})
+        }
+    })
+}
+
+const unfollowHandler = async (req, res) => {
+    const _id = req.params._id;
+    const userId = req.params.userId;
+    await client.connect();
+    const db = client.db('pang');
+    await db.collection('users').update({_id: _id}, {$pull : { followedBy : userId}}, (err, result) => {
+        if(result) {
+            console.log('working')
+            db.collection('users').update({_id: userId}, {$pull : { following : _id}}, (err, result) => {
+                if (result) {
```

Comment:
not sure I understand why you need to update 2x the same collection - isn't it possible in one operation ?